net.sourceforge.Lifeograph.json

```diff
@@ -0,0 +1,108 @@
+{
+    "app-id": "net.sourceforge.Lifeograph",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "lifeograph",
+    "finish-args": [ "--socket=x11", "--socket=wayland", "--filesystem=host" ],
+    "build-options" : {
```

Comment:
Those build options are already set by default.

Reply:
addressed in 06866a5dea7ad4fea7401422926a8a6ba70424db